docs/advanced-features/athena.md

```diff
@@ -137,20 +486,187 @@ SELECT
+    athena_await[{id}].s3_path: s3://mycompany-quilt-query-results/f38d2a58-dd81-43bf-a475-e6624f82e94d.txt
+    	athena_await f38d2a58-dd81-43bf-a475-e6624f82e94d.txt
+
+
+### B. Querying package-level metadata
 
 Suppose we wish to find all .tiff files produced by algorithm version 1.3
 with a cell index of 5.
+<!--pytest-codeblocks:cont-->
+
 
-```sql
-SELECT * FROM  "quilt_package_objects_YOUR_BUCKET_view" AS
+```python
+ATHENA_TEST = f"""
+SELECT * FROM {ATHENA_DB}.{OBJECTS_VIEW}
 WHERE substr(logical_key, -5)='.tiff'
 -- extract and query package-level metadata
 AND json_extract_scalar(meta, '$.user_meta.nucmembsegmentationalgorithmversion') LIKE '1.3%'
 AND json_array_contains(json_extract(meta, '$.user_meta.cellindex'), '5');
+"""
+```
+
+You can enter that Query directly in the Athena Query Editor using the QuiltWorkgroup, from the Queries -> Athena SQL tab, or using the following Python code:
+<!--pytest-codeblocks:cont-->
+
+
+```python
+print("\nTest Athena Query:")
+print(ATHENA_TEST)
+print("WorkGroup", ATHENA_WORKGROUP)
+results = athena_run(ATHENA_TEST)
+if results:
+    print("results")
+    print(results)
+```
+
+    
+    Test Athena Query:
+    
+    SELECT * FROM quilt_query.quilt_bio_products_quilt_objects_view
+    WHERE substr(logical_key, -5)='.tiff'
+    -- extract and query package-level metadata
+    AND json_extract_scalar(meta, '$.user_meta.nucmembsegmentationalgorithmversion') LIKE '1.3%'
+    AND json_array_contains(json_extract(meta, '$.user_meta.cellindex'), '5');
+    
+    WorkGroup quilt-query
+    	#9 athena_await[7c315a69-19af-4784-9299-bf2b020ad165]=QUEUED
+    	#8 athena_await[7c315a69-19af-4784-9299-bf2b020ad165]=RUNNING
+    	#7 athena_await[7c315a69-19af-4784-9299-bf2b020ad165]=FAILED
+    {'State': 'FAILED', 'StateChangeReason': 'com.amazonaws.services.s3.model.AmazonS3Exception: The specified bucket does not exist (Service: Amazon S3; Status Code: 404; Error Code: NoSuchBucket; Request ID: NWNXJA7Y7TMPFPD6; S3 Extended Request ID: zql9sBQ7JblYs9es43/xLSCMvh7H1WTFyNL4t7i1IfgATVyU4Hnx1Ya3nuvudMMpqEBgzP6DM2M=; Proxy: null), S3 Extended Request ID: zql9sBQ7JblYs9es43/xLSCMvh7H1WTFyNL4t7i1IfgATVyU4Hnx1Ya3nuvudMMpqEBgzP6DM2M= (Path: s3://quilt-bio-products/.quilt/packages)', 'SubmissionDateTime': datetime.datetime(2022, 8, 23, 8, 30, 16, 706000, tzinfo=tzlocal()), 'CompletionDateTime': datetime.datetime(2022, 8, 23, 8, 30, 18, 77000, tzinfo=tzlocal()), 'AthenaError': {'ErrorCategory': 2, 'ErrorType': 1306, 'Retryable': False, 'ErrorMessage': 'com.amazonaws.services.s3.model.AmazonS3Exception: The specified bucket does not exist (Service: Amazon S3; Status Code: 404; Error Code: NoSuchBucket; Request ID: NWNXJA7Y7TMPFPD6; S3 Extended Request ID: zql9sBQ7JblYs9es43/xLSCMvh7H1WTFyNL4t7i1IfgATVyU4Hnx1Ya3nuvudMMpqEBgzP6DM2M=; Proxy: null), S3 Extended Request ID: zql9sBQ7JblYs9es43/xLSCMvh7H1WTFyNL4t7i1IfgATVyU4Hnx1Ya3nuvudMMpqEBgzP6DM2M= (Path: s3://quilt-bio-products/.quilt/packages)'}}
```

Comment:
Is this supposed to be failed? If it is, I think it worth printing with [pprint()](https://docs.python.org/3/library/pprint.html#pprint.pprint).

Reply:
Adding print